### ISSUE TITLE
[Build Chain] Migrate to yarn + webpack

### ISSUE THREAD

**Issue by svenvanhal** [WISVCH/website#6]

**Comment by svenvanhal** [comment]
https://bower.io/blog/2017/how-to-migrate-away-from-bower/

**Comment by TimvdLippe** [comment]
What is the reasoning for adopting webpack? To replace `gulp`?

**Comment by svenvanhal** [comment]
Yes, and makes it a lot easier to support Foundation's new ES2016 javascript module structure. (https://zurb.com/blog/foundation-6-4-a-new-grid-javascript-upgrades-and-slick-new-prototyping-mode)
